arm-sql/2015-05-01-preview/swagger/failoverGroups.json

```diff
@@ -204,194 +253,155 @@
           }
         },
-        "deprecated": false
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "List failover group": {
+            "$ref": "../examples/FailoverGroupList.json"
+          }
+        }
       }
     },
     "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/failoverGroups/{failoverGroupName}/failover": {
       "post": {
         "tags": [
           "FailoverGroups"
         ],
-        "description": "Sets which server is primary by failing over from the current primary server.",
+        "description": "Fails over from the current primary server to this server.",
         "operationId": "FailoverGroups_Failover",
```

Comment:
operationId should be of the noun_verb form do consider changing this but keep in mind that would be a breaking change